server/elasticsearch.js

```diff
@@ -1,100 +1,23 @@
-const elasticSearch = require('elasticsearch');
-
-const elasticsearch = new elasticsearch.Client({
-	host: '127.0.0.1:9200',
-	log: 'info'
+var elasticsearch = require('elasticsearch');
```

Comment:
You should use 'const' here instead of 'var' since these variables are never redefined.